header.go

```diff
@@ -21,13 +21,13 @@ const (
 	cdInline     = "inline"
 
 	// Standard MIME content types
-	ctAppPrefix        = "application/"
+	ctAppPrefix        = "application/" //nolint
```

Comment:
@jhillyerd I need your decision on what to do with the unused code.

Reply:
I think we can just delete these, seems unlikely we will use them.

Reply:
Done